scripts/linux/runE2ETest.sh

```diff
@@ -447,19 +439,17 @@ function process_args() {
                 '-eventHubConsumerGroupId' ) saveNextArg=29;;
                 '-desiredModulesToRestartCSV' ) saveNextArg=30;;
                 '-restartIntervalInMins' ) saveNextArg=31;;
-                '-logAnalyticsEnabled' ) saveNextArg=32;;
-                '-logAnalyticsWorkspaceId' ) saveNextArg=33;;
-                '-logAnalyticsSharedKey' ) saveNextArg=34;;
-                '-logAnalyticsLogType' ) saveNextArg=35;;
-                '-twinUpdateSize' ) saveNextArg=36;;
-                '-twinUpdateFrequency' ) saveNextArg=37;;
-                '-twinUpdateFailureThreshold' ) saveNextArg=38;;
-                '-metricsEndpointsCSV' ) saveNextArg=39;;
-                '-metricsScrapeFrequencyInSecs' ) saveNextArg=40;;
-                '-metricsUploadTarget' ) saveNextArg=41;;
-                '-hostPlatform' ) saveNextArg=42;;
-                '-initializeWithAgentArtifact' ) saveNextArg=43;;
-                '-testInfo' ) saveNextArg=44;;
+                '-logAnalyticsWorkspaceId' ) saveNextArg=32;;
+                '-logAnalyticsSharedKey' ) saveNextArg=33;;
+                '-twinUpdateSize' ) saveNextArg=34;;
+                '-twinUpdateFrequency' ) saveNextArg=35;;
+                '-twinUpdateFailureThreshold' ) saveNextArg=36;;
+                '-metricsEndpointsCSV' ) saveNextArg=37;;
+                '-metricsScrapeFrequencyInSecs' ) saveNextArg=38;;
+                '-metricsUploadTarget' ) saveNextArg=39;;
+                '-hostPlatform' ) saveNextArg=40;;
+                '-initializeWithAgentArtifact' ) saveNextArg=41;;
+                '-testInfo' ) saveNextArg=42;;
                 '-cleanAll' ) CLEAN_ALL=1;;
```

Comment:
Clean up line 477-481; you may not be able to remove log analytics enabled flag, as it is needed to set false for e2e tests.